libc/test/src/math/smoke/LdExpTest.h

```diff
@@ -31,20 +31,31 @@ class LdExpTestTemplate : public LIBC_NAMESPACE::testing::FEnvSafeTest {
   const T nan = FPBits::quiet_nan().get_val();
 
   // A normalized mantissa to be used with tests.
-  static constexpr StorageType MANTISSA = NormalFloat::ONE + 0x1234;
+  static constexpr StorageType MANTISSA = NormalFloat::ONE + 0x123;
```

Comment:
`0x1234` is 13 bits, too large for `float16`'s 10-bit fraction.

(Comment moved from https://github.com/llvm/llvm-project/pull/94758.)